arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1110 @@
+        },
+        "sort": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SearchSort"
+          },
+          "description": "Gets or sets sort."
+        },
+        "requestTime": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The request time."
+        },
+        "aggregatedValueField": {
+          "type": "string",
+          "description": "The aggregated value field."
+        },
+        "aggregatedGroupingFields": {
+          "type": "string",
+          "description": "The aggregated grouping fields."
+        },
+        "sum": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The sum."
+        },
+        "max": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The max."
+        },
+        "schema": {
+          "$ref": "#/definitions/SearchMetadataSchema",
+          "description": "The schema."
+        }
+      },
+      "description": "Metadata for search results."
+    },
+    "SavedSearch": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the saved search."
+        },
+        "etag": {
+          "type": "string",
+          "description": "The etag of the saved search."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SavedSearchProperties",
+          "description": "Gets or sets properties of the saved search."
+        }
+      },
+      "description": "Value object for saved search results."
+    },
+    "SavedSearchesListResult": {
+      "properties": {
+        "__metadata": {
+          "x-ms-client-name": "Metadata",
+          "$ref": "#/definitions/SearchMetadata",
+          "description": "The metadata from search results."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SavedSearch"
+          },
+          "description": "The array of result values."
+        }
+      },
+      "description": "The saved search operation response."
+    },
+    "SearchError": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The error type."
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message."
+        }
+      },
+      "description": "Details for a search error."
+    },
+    "SearchResultsResponse": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the search, which includes the full url."
+        },
+        "__metadata": {
+          "x-ms-client-name": "Metadata",
+          "$ref": "#/definitions/SearchMetadata",
+          "description": "The metadata from search results."
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "object"
```

Comment:
Why is this of type "object" ? This would end up being of type JToken in C#. Shouldn't there be a model definition named "SearchResult" that has well defined model properties ?